src/rest_utils.hpp

```diff
@@ -20,6 +20,8 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wall"
 #include "tensorflow_serving/apis/prediction_service.grpc.pb.h"
+
+#include "src/kfserving_api/grpc_predict_v2.grpc.pb.h"
```

Comment:
I think it is not necessary to have it inside pragma clause

Reply:
Additionally it is not necessary to have it at all in header. Just forward declare & include in implementation file
```suggestion
namespace inference {
class ModelInferResponse;
}
```